client/signup/steps/about/index.jsx

```diff
@@ -93,14 +98,13 @@ class AboutStep extends Component {
 		this.setState( { query: '' } );
 	};
 
-	handleSuggestionChangeEvent = event => {
-		this.setState( { query: event.target.value } );
-		this.setState( { siteTopicValue: event.target.value } );
+	handleSuggestionChangeEvent = ( { target: { name, value } } ) => {
+		this.setState( { query: value } );
+		this.setState( { siteTopicValue: value } );
```

Comment:
is there a reason we aren't setting these together? or of sequencing the following updates until after setting the state?

```js
handleSuggestionChangeEvent = ( { target: { name, value } } ) => this.setState( {
	query: value,
	siteTopicValue: value,
}, () => {
	this.props.recordTracksEvent( 'calypso_signup_actions_select_site_topic', { value } ),
	this.formStateController.handleFieldChange( { name, value } );
} );
```

Reply:
In hindsight, your suggested change would be much better. I'll look into putting this change in when we rip out the A/B test.